python/ray/tests/test_log_dedup.py

```diff
@@ -19,6 +19,41 @@ def test_nodedup_logs_single_process():
     assert out1 == [batch1]
 
 
+def test_nodedup_logs_only_canonicalized_lines():
+    now = 142300000.0
+
+    def gettime():
+        return now
+
+    dedup = LogDeduplicator(5, None, None, _timesource=gettime)
+    batch1 = {
+        "ip": "node1",
+        "pid": 100,
+        # numbers are canonicalised, so this would lead to empty dedup_key
+        "lines": ["1"],
```

Comment:
since we are testing `LogDeduplicator` can you add more duplicate lines? like `2` then `2` and all are outputed.

Reply:
Done